numba/core/ssa.py

```diff
@@ -65,16 +80,16 @@ def _run_ssa(blocks):
     return blocks
 
 
-def _fix_ssa_vars(blocks, varname, defmap, cfg, df_plus):
+def _fix_ssa_vars(blocks, varname, defmap, cfg, df_plus, cache_list_vars):
     """Rewrite all uses to ``varname`` given the definition map
     """
     states = _make_states(blocks)
     states['varname'] = varname
     states['defmap'] = defmap
     states['phimap'] = phimap = defaultdict(list)
-    states['cfg'] = cfg = compute_cfg_from_blocks(blocks)
```

Comment:
NOTE: I think the logic change here is to use the CFG from the top of `_run_ssa` (and `df+` computed from that) as there's nothing that should alter the CFG or these relations prior to computing the phi's? This should save some CFG building.

Reply:
Yes, nothing is changing the CFG. we don't have to recompute this per variable

Reply:
Great, thanks for confirming that my understanding was correct.